src/ethers/signer.ts

```diff
@@ -50,16 +52,84 @@ function restoreCustomFields(tx: Deferrable<TransactionRequest>, savedFields: an
   }
 }
 
-// Wallet API dynamic hooking
-// Wallet.populateTransaction = function() {
-// };
 
-export class KlaytnWallet extends Wallet {
+export class Wallet extends OldWallet {
 
-  checkTransaction(transaction: Deferrable<TransactionRequest>): Deferrable<TransactionRequest> {
+  private klaytn_address: string | undefined;
+  
+  private dynamicUpdateWalletAPI;
+  
+  _checkTransaction:((transaction: Deferrable<TransactionRequest>) => Deferrable<TransactionRequest>) | undefined ;
+  _populateTransaction: ((transaction: Deferrable<TransactionRequest>) => Promise<TransactionRequest>) | undefined ;
+
+  _signTransaction; 
+  _sendTransaction;
+
+  constructor(address: any, privateKey?: any, provider?: Provider, dynamicUpdateWalletAPI: boolean=true ) {
+    let str_addr = String(address); 
+
+    if ( HexStr.isHex(address) && str_addr.length == 42 && str_addr.startsWith("0x")) {
```

Comment:
Use ethers.utils.getAddress to normalize address https://docs.ethers.org/v5/api/utils/address/#utils-getAddress